src/main/java/br/unb/cic/analysis/df/Collector.java

```diff
@@ -1,25 +1,48 @@
 package br.unb.cic.analysis.df;
 
 import br.unb.cic.analysis.model.Conflict;
+import soot.jimple.StaticFieldRef;
+import soot.jimple.internal.JInstanceFieldRef;
 
+import java.util.HashMap;
 import java.util.HashSet;
 import java.util.Set;
 
 class Collector {
 
     private Set<Conflict> conflicts;
+    private Set<HashMap<String, JInstanceFieldRef>> hashMap;
```

Comment:
Just like "hashMapStatic" I think you could rename it to "hashMapJInstanceField" or something, to make it clearer what you're talking about ...

Is it not possible to use only one hash map for analysis?

Reply:
Very good. I will do this.